### PR TITLE
Add DrawingContainer

### PR DESCRIPTION
**Pull request by mcbouslog**
Package:
- lib-classifier
- helpful staging workflow: http://localhost:8080/?project=1876&workflow=3344

Staging project and workflow:
- http://localhost:8080/?project=1876&workflow=3344

Describe your changes:
- changes InteractionLayer event handlers from onMouse to onPointer
- refactors drawing store to more explicit naming
- renders DrawingContainer in InteractionLayerContainer if drawing task in active workflow step
- adds DrawingContainer for current and active workflow step markings
- adds Point and Line (and Root) drawing tool components

Notes:
- these changes are a portion of #1078 
- subsequent to these changes, adding the coordinate stream (as in #1078) and tool stores, saving markings to annotation (#1103), read-only render for previous markings (#1104) and delete marking (#1106) can all be worked on concurrently
- there is some temporary code to randomly add a point or line on `pointerup` to allow the items noted above to be worked on concurrently

Why WIP:
- [x] DrawingContainer tests
- [x] Root tests
- [x] Line tests

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && yarn bootstrap` and app works as expected?
- [x] Can you run a [production build](https://github.com/zooniverse/front-end-monorepo#getting-started) of the app?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    }
+    if (tool.type === 'point') {
+      const coordinates = { x: (Math.floor(Math.random() * 400)), y: (Math.floor(Math.random() * 400)) }
+      this.finishMark(coordinates)
+    }
+  }
+
+  setActiveMark () {
+    const { activeDrawingTool, activeStepTasks } = this.props
+    const [activeDrawingTask] = activeStepTasks.filter(task => task.type === 'drawing')
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js]: I think we discussed having the drawing store filter this on a view getter function? Totally ok if that's going to come in a later PR.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/tools/Root.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { expect } from 'chai'
+import Root from './Root'
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/tools/Root.spec.js]: Non-blocking and more of an FYI. I think with the new module spec (mjs) in javascript that if the export name and the filename do not match that it'll error. So, we should try to match the file names to the export name whenever possible. Maybe rename the file to `DrawingToolRoot`?

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js (hunk 1 above)
Comment (mcbouslog) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js]: Yes and yes, those changes are in PR #1164, [here's a link specifically to those views in the store](https://github.com/zooniverse/front-end-monorepo/pull/1164/files#diff-e76319fbfa25691959816b6c45adbf06R11).

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+    this.setState({ activeMark: newMark, marks: newMarks })
+  }
+
+  render () {
+    const { activeMark, marks } = this.state
+    if (!activeMark) return null
+
+    const MarkComponent = getDrawingTool(activeMark.tool.type)
+    // TODO: add error handling if don't get requested tool type
```
Comment (simensta) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js]: @mcbouslog, should we address this to do in a separate PR? If so, I can create issue for https://github.com/zooniverse/front-end-monorepo/projects/11.

Comment (mcbouslog) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/DrawingContainer.js]: I think punt for now? I'm not sure exactly what behavior we'd like, i.e. console warning prob at minimum, but do we also want something in UI?

**Comment by srallen** [comment]
@simensta are those to dos noted in an issue somewhere so we don't lose track of them?

**Comment by eatyourgreens** [comment]
Check out the daily classification chart for focusable SVG.

**Pull request merged**